File: 2026POTD/JANUARY/26-01-2026/1200.swift
class Solution {
    func minimumAbsDifference(_ arr: [Int]) -> [[Int]] {
        let a = arr.sorted()
        var mn = Int.max
        for i in 1..<a.count {
            mn = min(mn, a[i] - a[i - 1])
        }
        var res = [[Int]]()
        for i in 1..<a.count {
            if a[i] - a[i - 1] == mn {
                res.append([a[i - 1], a[i]])
            }
        }
        return res
    }
}

File: 2026POTD/JANUARY/26-01-2026/1200.php
class Solution {
    function minimumAbsDifference($arr) {
        sort($arr);
        $mn = PHP_INT_MAX;
        for ($i = 1; $i < count($arr); $i++)
            $mn = min($mn, $arr[$i] - $arr[$i - 1]);

        $res = [];
        for ($i = 1; $i < count($arr); $i++)
            if ($arr[$i] - $arr[$i - 1] == $mn)
                $res[] = [$arr[$i - 1], $arr[$i]];
        return $res;
    }
}